src/main/java/com/knu/buga1chuk/serialization/service/JsonPersonSerializationService.java

```diff
@@ -54,21 +64,23 @@ public void writePersons(File file, PersonList persons) {
 
     }
 
-    public String toJson(Person person) {
+    @Override
+    public String toString(PersonList personList) {
         Gson gson = new Gson();
-        return gson.toJson(person);
+        return gson.toJson(personList);
     }
 
-    public Person toPerson(String personJson) {
-
+    @Override
+    public PersonList toPersonList(String personListJson) {
```

Comment:
What if somebody pass 'null' as a parameter.
The following line will fail:
```
        if (!personListJson.equals("")) {
```

Reply:
fix

Reply:
Maybe it is better to do it like this:
```
if (!"".equals(personListJson)) {}
```

Reply:
fix